debugging/riscv_debug_generator/Cargo.toml

```diff
@@ -0,0 +1,25 @@
+[package]
+name = "riscv_debug_generator"
+version = "0.1.0"
+authors = ["Kai Meinhard <kaimeinhard@hotmail.de>"]
+edition = "2018"
+
+[lib]
+crate-type = ["staticlib", "rlib"]
+
+[profile.release]
+opt-level = 3
+codegen-units = 1
+lto = "fat"
+
+[dependencies]
+# TODO this is an out of repo path
+tapasco = { path = "../../../tapasco/runtime/libtapasco" }
```

Comment:
You can directly link to the git repository here:
`tapasco = { git = "git@github.com:esa-tu-darmstadt/tapasco.git" }`, see https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html#specifying-dependencies-from-git-repositories for details.

Reply:
Thank you for the info! I wasn't aware of that.